### PR TITLE
RELATED: RAIL-1553 add ExtendedDateFiltes types

### PR DESCRIPTION
**Pull request by no23reason**

### CODE REVIEW

**Comment by yenkins** [comment]
Currently destined for [release R182](https://confluence.intgdc.com/display/Dev/R182).
This release is **zero downtime** release.
Code drop: 2019-05-18 00:00.

**Comment by yenkins** [comment]
Sub-task [RAIL-1553](http://jira.intgdc.com/browse/RAIL-1553) Move Extended Date Filters types to typings package
	Status: In Progress
	Priority: Major
	Assignee: Dan Homola

**Comment by yenkins** [comment]
## client-libs/gooddata-typings-pull-request-compliance-check check failed
### prohibit-subtask-ticket
Do not reference subtask in commit message summary. It introduces risk of undetected partial changes.
- RAIL-1553
### require-evaluated-risk
Ticket should have regression risk evaluated before merging, see [GoodConfluence](https://confluence.intgdc.com/display/office/Labels+In+JIRA#LabelsInJIRA-QA&DELI)
- RAIL-1553

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/11/ : UNSTABLE in 15s
- gooddata-typings-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by yenkins** [comment]
Story [RAIL-1531](http://jira.intgdc.com/browse/RAIL-1531) Loading Date Filter Configuration from Project Metadata
	Status: New
	Priority: Major
	Assignee: Rail Guild User

**Comment by yenkins** [comment]
## client-libs/gooddata-typings-pull-request-compliance-check check failed
### require-evaluated-risk
Ticket should have regression risk evaluated before merging, see [GoodConfluence](https://confluence.intgdc.com/display/office/Labels+In+JIRA#LabelsInJIRA-QA&DELI)
- RAIL-1531

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/12/ : UNSTABLE in 20s
- gooddata-typings-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by no23reason** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/13/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/79/ : FAILURE in 46s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/14/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/80/ : SUCCESS in 2m 39s

File: src/tests/ExtendedDateFilters.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        visible: true
+    };
+
+    const allTimeFilter: ExtendedDateFilters.IAllTimeDateFilter = {
+        localIdentifier: 'foo',
+        type: 'allTime'
+    };
+
+    describe('isAbsoluteDateFilterOption', () => {
+        it.each([
```
Comment (tomasreichmann) [inline src/tests/ExtendedDateFilters.ts]: could be merged with true

Comment (no23reason) [reply src/tests/ExtendedDateFilters.ts]: It could, but then we would have 4 `false`s and only 1 `true` which seems not better

**Comment by yenkins** [comment]
## client-libs/gooddata-typings-pull-request-compliance-gate check failed
### require-work-in-progress
Ticket has to be in progress: Hacking, Verification
- RAIL-1531 - New

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/5/ : UNSTABLE in 12s
- gooddata-typings-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/6/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/81/ : SUCCESS in 30s

**Pull request merged**